torchtext/vocab.py

```diff
@@ -31,7 +31,7 @@ class Vocab(object):
     # TODO (@mttk): Populate classs with default values of special symbols
     UNK = '<unk>'
 
-    def __init__(self, counter, max_size=None, min_freq=1, specials=['<unk>', '<pad>'],
+    def __init__(self, counter, max_size=None, min_freq=1, specials=('<unk>', '<pad>'),
```

Comment:
[Mutable default arguments](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments):

"""
A new list is created once when the function is defined, and the same list is used in each successive call.

Python’s default arguments are evaluated once when the function is defined, not each time the function is called (like it is in say, Ruby). This means that if you use a mutable default argument and mutate it, you will and have mutated that object for all future calls to the function as well.
"""

We could use None as a default and construct a list if it is None, but I don't see the need for ```specials``` to be a list based on the rest of the code. [A tuple is not mutable](https://docs.python.org/3.8/library/stdtypes.html#tuples), so we can use it instead.